kotlinpoet/src/main/java/com/squareup/kotlinpoet/LambdaTypeName.kt

```diff
@@ -17,8 +17,11 @@ package com.squareup.kotlinpoet
 
 import kotlin.reflect.KClass
 
+@OptIn(ContextReceivers::class)
 public class LambdaTypeName private constructor(
   public val receiver: TypeName? = null,
+  @property:ContextReceivers
+  public val contextReceivers: List<TypeName>? = emptyList(),
```

Comment:
We should pick null or empty but not both

Reply:
Looks like an oversight on my part, it was supposed to be not nullable and default to empty